### DISCUSSION TITLE
Pytorch v1.0 整体进度（校对+官方纪念品）

### DISCUSSION THREAD

**Discussion by wizardforcel** [apachecn/pytorch-doc-zh#618] [Announcements]
> 官方奖励（好消息 好消息）

凡事翻译过贡献者，都可以申请得到官方的纪念礼物：
如果有问题的人私聊： QQ 529815144

```
邮件部分内容：
As a start, let us know where we can ship you some PyTorch swag! 
We have stickers, t-shirts, hoodies, and backpacks – let us know what sizes you need.
```

官方认领礼物地址如下: https://github.com/apachecn/pytorch-doc-zh/issues/353

> 认领须知

最新版本 PyTorch v1.2 校验任务认领地址 - <https://github.com/apachecn/pytorch-doc-zh/issues/422>

**提交的时候不要改动文件名称，即使它跟章节标题不一样也不要改，因为文件名和原文的链接是对应的！！！**

留言格式：昵称 + QQ + 章节

需要取消认领的也在此留言。

| 章节   | 校验者 | 进度 |
| --- | --- | --- |
| 教程部分 | - | -|
| [Deep Learning with PyTorch: A 60 Minute Blitz](https://github.com/apachecn/pytorch-doc-zh/blob/master/docs/1.0/deep_learning_60min_blitz.md) | @FontTian | 100% |
| [What is PyTorch?](https://github.com/apachecn/pytorch-doc-zh/blob/master/docs/1.0/blitz_tensor_tutorial.md) | @FontTian | 100% |
| [Autograd: Automatic Differentiation](https://github.com/apachecn/pytorch-doc-zh/blob/master/docs/1.0/blitz_autograd_tutorial.md) | @FontTian | 100% |
| [Neural Networks](https://github.com/apachecn/pytorch-doc-zh/blob/master/docs/1.0/blitz_neural_networks_tutorial.md) |@FontTian | 100% |
| [Training a Classifier](https://github.com/apachecn/pytorch-doc-zh/blob/master/docs/1.0/blitz_cifar10_tutorial.md) |@FontTian | 100% |
| [Optional: Data Parallelism](https://github.com/apachecn/pytorch-doc-zh/blob/master/docs/1.0/blitz_data_parallel_tutorial.md) | @FontTian | 100% |
| [Data Loading and Processing Tutorial](https://github.com/apachecn/pytorch-doc-zh/blob/master/docs/1.0/data_loading_tutorial.md) | @dyywinner | 100% |
| [Learning PyTorch with Examples](https://github.com/apachecn/pytorch-doc-zh/blob/master/docs/1.0/pytorch_with_examples.md) | @wulongjian | |
| [Transfer Learning Tutorial](https://github.com/apachecn/pytorch-doc-zh/blob/master/docs/1.0/transfer_learning_tutorial.md) | @infdahai  | 100% |
| [Deploying a Seq2Seq Model with the Hybrid Frontend](https://github.com/apachecn/pytorch-doc-zh/blob/master/docs/1.0/deploy_seq2seq_hybrid_frontend_tutorial.md) | @FontTian | 100% |
| [Saving and Loading Models](https://github.com/apachecn/pytorch-doc-zh/blob/master/docs/1.0/saving_loading_models.md) | @luxinfeng | |
| [What is torch.nn really?](https://github.com/apachecn/pytorch-doc-zh/blob/master/docs/1.0/nn_tutorial.md) | @luxinfeng | |
| [Finetuning Torchvision Models](https://github.com/apachecn/pytorch-doc-zh/blob/master/docs/1.0/finetuning_torchvision_models_tutorial.md) | @luxinfeng | |
| [Spatial Transformer Networks Tutorial](https://github.com/apachecn/pytorch-doc-zh/blob/master/docs/1.0/spatial_transformer_tutorial.md) | | |
| [Neural Transfer Using PyTorch](https://github.com/apachecn/pytorch-doc-zh/blob/master/docs/1.0/neural_style_tutorial.md) | @AllenZYJ | |
| [Adversarial Example Generation](https://github.com/apachecn/pytorch-doc-zh/blob/master/docs/1.0/fgsm_tutorial.md) | | |
| [Transfering a Model from PyTorch to Caffe2 and Mobile using ONNX](https://github.com/apachecn/pytorch-doc-zh/blob/master/docs/1.0/super_resolution_with_caffe2.md) | | |
| [Chatbot Tutorial](https://github.com/apachecn/pytorch-doc-zh/blob/master/docs/1.0/chatbot_tutorial.md) | @jiangzhonglian | 100% |
| [Generating Names with a Character-Level RNN](https://github.com/apachecn/pytorch-doc-zh/blob/master/docs/1.0/char_rnn_generation_tutorial.md) | @hijkzzz | 100% |
| [Classifying Names with a Character-Level RNN](https://github.com/apachecn/pytorch-doc-zh/blob/master/docs/1.0/char_rnn_classification_tutorial.md) | @hijkzzz | |
| [Deep Learning for NLP with Pytorch](https://github.com/apachecn/pytorch-doc-zh/blob/master/docs/1.0/deep_learning_nlp_tutorial.md) | @FontTian | 100%  |
| [Introduction to PyTorch](https://github.com/apachecn/pytorch-doc-zh/blob/master/docs/1.0/nlp_pytorch_tutorial.md) | @FontTian | 100%  |
| [Deep Learning with PyTorch](https://github.com/apachecn/pytorch-doc-zh/blob/master/docs/1.0/nlp_deep_learning_tutorial.md) | @FontTian | 100% |
| [Word Embeddings: Encoding Lexical Semantics](https://github.com/apachecn/pytorch-doc-zh/blob/master/docs/1.0/nlp_word_embeddings_tutorial.md) | | |
| [Sequence Models and Long-Short Term Memory Networks](https://github.com/apachecn/pytorch-doc-zh/blob/master/docs/1.0/nlp_sequence_models_tutorial.md) | @FontTian | 100%  |
| [Advanced: Making Dynamic Decisions and the Bi-LSTM CRF](https://github.com/apachecn/pytorch-doc-zh/blob/master/docs/1.0/nlp_advanced_tutorial.md) | @enningxie | 100% |
| [Translation with a Sequence to Sequence Network and Attention](https://github.com/apachecn/pytorch-doc-zh/blob/master/docs/1.0/seq2seq_translation_tutorial.md) | @FontTian  | 100% |
| [DCGAN Tutorial](https://github.com/apachecn/pytorch-doc-zh/blob/master/docs/1.0/dcgan_faces_tutorial.md) | @jiangzhonglian | 100% |
| [Reinforcement Learning (DQN) Tutorial](https://github.com/apachecn/pytorch-doc-zh/blob/master/docs/1.0/reinforcement_q_learning.md) |  | |
| [Creating Extensions Using numpy and scipy](https://github.com/apachecn/pytorch-doc-zh/blob/master/docs/1.0/numpy_extensions_tutorial.md) | @FontTian | 100% |
| [Custom C++   and CUDA Extensions](https://github.com/apachecn/pytorch-doc-zh/blob/master/docs/1.0/cpp_extension.md) | | |
| [Extending TorchScript with Custom C++   Operators](https://github.com/apachecn/pytorch-doc-zh/blob/master/docs/1.0/torch_script_custom_ops.md) | | |
| [Writing Distributed Applications with PyTorch](https://github.com/apachecn/pytorch-doc-zh/blob/master/docs/1.0/dist_tuto.md) | | |
| [PyTorch 1.0 Distributed Trainer with Amazon AWS](https://github.com/apachecn/pytorch-doc-zh/blob/master/docs/1.0/aws_distributed_training_tutorial.md) | | |
| [ONNX Live Tutorial](https://github.com/apachecn/pytorch-doc-zh/blob/master/docs/1.0/ONNXLive.md) | | |
| [Loading a PyTorch Model in C++](https://github.com/apachecn/pytorch-doc-zh/blob/master/docs/1.0/cpp_export.md) | | |
| [Using the PyTorch C++   Frontend](https://github.com/apachecn/pytorch-doc-zh/blob/master/docs/1.0/cpp_frontend.md) | | |
| 文档部分   | - | -|
| [Autograd mechanics](https://github.com/apachecn/pytorch-doc-zh/blob/master/docs/1.0/notes_autograd.md) | @AlexJakin | 100% |
| [Broadcasting semantics](https://github.com/apachecn/pytorch-doc-zh/blob/master/docs/1.0/notes_broadcasting.md) | @AlexJakin | 100% |
| [CUDA semantics](https://github.com/apachecn/pytorch-doc-zh/blob/master/docs/1.0/notes_cuda.md) | @AlexJakin | |
| [Extending PyTorch](https://github.com/apachecn/pytorch-doc-zh/blob/master/docs/1.0/notes_extending.md) | | |
| [Frequently Asked Questions](https://github.com/apachecn/pytorch-doc-zh/blob/master/docs/1.0/notes_faq.md) | | |
| [Multiprocessing best practices](https://github.com/apachecn/pytorch-doc-zh/blob/master/docs/1.0/notes_multiprocessing.md) | | |
| [Reproducibility](https://github.com/apachecn/pytorch-doc-zh/blob/master/docs/1.0/notes_randomness.md) | @bruce1408 | |
| [Serialization semantics](https://github.com/apachecn/pytorch-doc-zh/blob/master/docs/1.0/notes_serialization.md) | | |
| [Windows FAQ](https://github.com/apachecn/pytorch-doc-zh/blob/master/docs/1.0/notes_windows.md) | | |
| [torch](https://github.com/apachecn/pytorch-doc-zh/blob/master/docs/1.0/torch.md) |  | | |
| [Tensors](https://github.com/apachecn/pytorch-doc-zh/blob/master/docs/1.0/torch_tensors.md) | @dyywinner  | |
| [Random sampling](https://github.com/apachecn/pytorch-doc-zh/blob/master/docs/1.0/torch_random_sampling.md) | | |
| [Serialization, Parallelism, Utilities](https://github.com/apachecn/pytorch-doc-zh/blob/master/docs/1.0/torch_serialization_parallelism_utilities.md) | | |
| [Pointwise Ops](https://github.com/apachecn/pytorch-doc-zh/blob/master/docs/1.0/torch_math_operations_pointwise_ops.md) | | |
| [Reduction Ops](https://github.com/apachecn/pytorch-doc-zh/blob/master/docs/1.0/torch_math_operations_reduction_ops.md) | | |
| [Comparison Ops](https://github.com/apachecn/pytorch-doc-zh/blob/master/docs/1.0/torch_math_operations_comparison_ops.md) | | |
| [Spectral Ops](https://github.com/apachecn/pytorch-doc-zh/blob/master/docs/1.0/torch_math_operations_spectral_ops.md) | | |
| [Other Operations](https://github.com/apachecn/pytorch-doc-zh/blob/master/docs/1.0/torch_math_operations_other_ops.md) | | |
| [BLAS and LAPACK Operations](https://github.com/apachecn/pytorch-doc-zh/blob/master/docs/1.0/torch_math_operations_blas_lapack_ops.md) | | |
| [torch.Tensor](https://github.com/apachecn/pytorch-doc-zh/blob/master/docs/1.0/tensors.md) | | |
| [Tensor Attributes](https://github.com/apachecn/pytorch-doc-zh/blob/master/docs/1.0/tensor_attributes.md) | | |
| [Type Info](https://github.com/apachecn/pytorch-doc-zh/blob/master/docs/1.0/type_info.md) | | |
| [torch.sparse](https://github.com/apachecn/pytorch-doc-zh/blob/master/docs/1.0/sparse.md) | | |
| [torch.cuda](https://github.com/apachecn/pytorch-doc-zh/blob/master/docs/1.0/cuda.md) | | |
| [torch.Storage](https://github.com/apachecn/pytorch-doc-zh/blob/master/docs/1.0/storage.md) | | |
| [torch.nn](https://github.com/apachecn/pytorch-doc-zh/blob/master/docs/1.0/nn.md) | | |
| [torch.nn.functional](https://github.com/apachecn/pytorch-doc-zh/blob/master/docs/1.0/nn_functional.md) | @Originval | 100% |
| [torch.nn.init](https://github.com/apachecn/pytorch-doc-zh/blob/master/docs/1.0/nn_init.md) | @dyywinner | |
| [torch.optim](https://github.com/apachecn/pytorch-doc-zh/blob/master/docs/1.0/optim.md) | @zonasw | |
| [Automatic differentiation package - torch.autograd](https://github.com/apachecn/pytorch-doc-zh/blob/master/docs/1.0/autograd.md) | | |
| [Distributed communication package - torch.distributed](https://github.com/apachecn/pytorch-doc-zh/blob/master/docs/1.0/distributed.md) | | |
| [Probability distributions - torch.distributions](https://github.com/apachecn/pytorch-doc-zh/blob/master/docs/1.0/distributions.md) | | |
| [Torch Script](https://github.com/apachecn/pytorch-doc-zh/blob/master/docs/1.0/jit.md) | | |
| [Multiprocessing package - torch.multiprocessing](https://github.com/apachecn/pytorch-doc-zh/blob/master/docs/1.0/multiprocessing.md) | | |
| [torch.utils.bottleneck](https://github.com/apachecn/pytorch-doc-zh/blob/master/docs/1.0/bottleneck.md) | | |
| [torch.utils.checkpoint](https://github.com/apachecn/pytorch-doc-zh/blob/master/docs/1.0/checkpoint.md) | | |
| [torch.utils.cpp_extension](https://github.com/apachecn/pytorch-doc-zh/blob/master/docs/1.0/docs_cpp_extension.md) | | |
| [torch.utils.data](https://github.com/apachecn/pytorch-doc-zh/blob/master/docs/1.0/data.md) | | |
| [torch.utils.dlpack](https://github.com/apachecn/pytorch-doc-zh/blob/master/docs/1.0/dlpack.md) | | |
| [torch.hub](https://github.com/apachecn/pytorch-doc-zh/blob/master/docs/1.0/hub.md) | | |
| [torch.utils.model_zoo](https://github.com/apachecn/pytorch-doc-zh/blob/master/docs/1.0/model_zoo.md) | | |
| [torch.onnx](https://github.com/apachecn/pytorch-doc-zh/blob/master/docs/1.0/onnx.md) | | |
| [Distributed communication package (deprecated) - torch.distributed.deprecated](https://github.com/apachecn/pytorch-doc-zh/blob/master/docs/1.0/distributed_deprecated.md) | | |
| [torchvision Reference](https://github.com/apachecn/pytorch-doc-zh/blob/master/docs/1.0/docs_torchvision_ref.md) | | |
| [torchvision.datasets](https://github.com/apachecn/pytorch-doc-zh/blob/master/docs/1.0/torchvision_datasets.md) | | |
| [torchvision.models](https://github.com/apachecn/pytorch-doc-zh/blob/master/docs/1.0/torchvision_models.md) | | |
| [torchvision.transforms](https://github.com/apachecn/pytorch-doc-zh/blob/master/docs/1.0/torchvision_transforms.md) | | |
| [torchvision.utils](https://github.com/apachecn/pytorch-doc-zh/blob/master/docs/1.0/torchvision_utils.md) | | |

**Comment by jiangzhonglian** [comment]
最新版本 PyTorch v1.2 校验任务认领地址 - <https://github.com/apachecn/pytorch-doc-zh/issues/422>

**Comment by AllenZYJ** [comment]
Neural Networks + 张一极 + 2533524298

**Comment by friedhelm739** [comment]
Reinforcement Learning (DQN) Tutorial + 平淡的天+ 554650680

**Comment by wangshuai9517** [comment]
DCGAN Tutorial + 王帅 +946560713

**Comment by a625687551** [comment]
Chatbot Tutorial +毛毛虫+625687551

**Comment by hhxx2015** [comment]
Classifying Names with a Character-Level RNN + hhxx2015 + 1347261894

**Comment by wangshuai9517** [comment]
> DCGAN Tutorial + 王帅 +946560713

用户名是：wangshuai9517

**Comment by fanqie03** [comment]
What is PyTorch? + mengfu188 +1242733702

**Comment by badbubble** [comment]
Learning PyTorch with Examples QQ:675972642

**Comment by sight007** [comment]
Training a Classifier   万华镜  QQ：249053957

**Comment by bat67** [comment]
原来出于学习和兴趣翻译过一些~  所以想直接带上github的链接搬移过来~
QQ：244596167

Deep Learning with PyTorch: A 60 Minute Blitz
* What is PyTorch?
* Autograd: Automatic Differentiation
* Neural Networks
* Training a Classifier
* Optional: Data Parallelism

> 链接：https://github.com/bat67/Deep-Learning-with-PyTorch-A-60-Minute-Blitz-cn

Learning PyTorch with Examples

> 链接：https://github.com/bat67/pytorch-examples-cn

**Comment by hiyongz** [comment]
Finetuning Torchvision Models + 1028740976

**Comment by cangyunye** [comment]
Deploying a Seq2Seq Model with the Hybrid Frontend + 421524799
Adversarial Example Generation + 421524799 
用户名：cangyunye

**Comment by talengu** [comment]
Loading a PyTorch Model in C++
523125112

**Comment by Watermelon233** [comment]
> Neural Networks + 张一极 + 2533524298

大佬 已经有人认领了此章节  要不要换一章~

**Comment by guobaoyo** [comment]
Introduction to PyTorch+YAOKE7+1415500736

**Comment by badbubble** [comment]
Sequence Models and Long-Short Term Memory Networks + ^^^ + 675972642

**Comment by fanqie03** [comment]
Translation with a Sequence to Sequence Network and Attention + mengfu188 +1242733702

**Comment by wizardforcel** [comment]
@AllenZYJ @sight007 

大佬 已经有人认领了此章节 要不要换一章~

**Comment by bdqfork** [comment]
Deep Learning with PyTorch and Neural Transfer Using PyTorch + 760751140

**Comment by firdameng** [comment]
Writing Distributed Applications with PyTorch + 1546233608

**Comment by yportne13** [comment]
Data Loading and Processing Tutorial + 406681405

**Comment by cjf1699** [comment]
Data Loading and Processing Tutorial+954467538

**Comment by lhc741** [comment]
What is <cite>torch.nn</cite> really? + 568547745

**Comment by sight007** [comment]
Word Embeddings: Encoding Lexical Semantics   +  249053957  如果没人做的话

**Comment by solerji** [comment]
Using the PyTorch C++ Frontend+solerji+2474499153

**Comment by cangyunye** [comment]
Creating Extensions Using numpy and scipy + cangyunye + 421524799

**Comment by PEGASUS1993** [comment]
Spatial Transformer Networks Tutorial  + 冯宝宝 + 2548541640

**Comment by PEGASUS1993** [comment]
ONNX Live Tutorial  这个我也占了，不接受反驳，哈哈哈~~~

**Comment by PEGASUS1993** [comment]
Transfering a Model from PyTorch to Caffe2 and Mobile using ONNX 我也占了 ，目前翻译了一大半。(PS：本来也想翻译PyTorch 1.0 Distributed Trainer with Amazon AWS，翻译了一小部分，感觉有点费劲，不知所云。)

**Comment by jiangzhonglian** [comment]
CUDA semantics + 片刻 + 529815144

**Comment by guobaoyo** [comment]
torch.onnx+guobaoyo+1415500736

**Comment by PEGASUS1993** [comment]
Autograd mechanics，Broadcasting semantics，Extending PyTorch，Frequently Asked Questions，Windows FAQ，Type Info。 这六个我都占了，争取能节前翻译完第一版。 冯宝宝/PEGASUS1993 + 2548541640

**Comment by bdqfork** [comment]
torch.cuda + 760751140

**Comment by cvley** [comment]
Multiprocessing best practices  + 873596808

**Comment by yportne13** [comment]
PyTorch 1.0 Distributed Trainer with Amazon AWS+406681405

**Comment by hijkzzz** [comment]
torch.Tensor + hijkzzz + 125036525
过年在家翻译完

**Comment by hijkzzz** [comment]
torch.nn.functional + hijkzzz + 125036525
再翻译一章

**Comment by hijkzzz** [comment]
Multiprocessing package - torch.multiprocessing + hijkzzz + 125036525
OK 搞完这个就开心打游戏

**Comment by GeneZC** [comment]
torch.nn.init + 654211094

**Comment by hijkzzz** [comment]
sparse.md + hijkzzz + 125036525

**Comment by hijkzzz** [comment]
Probability distributions - torch.distributions + hijkzzz + 125036525

**Comment by gfjiangly** [comment]
Automatic differentiation package - torch.autograd
2331262129

**Comment by yuange250** [comment]
Tensor Attributes + 阿远 + 1285714007

**Comment by yuange250** [comment]
torch.Storage + 1285714007

**Comment by BXuan694** [comment]
torchvision Reference+2490278771

**Comment by BXuan694** [comment]
torchvision.datasets+2490278771

**Comment by BXuan694** [comment]
torchvision.models+2490278771

**Comment by BXuan694** [comment]
torchvision.transforms+2490278771

**Comment by BXuan694** [comment]
torchvision.utils+2490278771

**Comment by yuange250** [comment]
Serialization semantics + 1285714007

**Comment by yuange250** [comment]
torch.nn + 1285714007

**Comment by BXuan694** [comment]
torch.utils.model_zoo+2490278771

**Comment by BXuan694** [comment]
torch.utils.data+2490278771

**Comment by intHbl** [comment]
- [torch.utils.bottleneck](https://github.com/apachecn/pytorch-doc-zh/blob/master/docs/1.0/bottleneck.md)		
- [torch.utils.checkpoint](https://github.com/apachecn/pytorch-doc-zh/blob/master/docs/1.0/checkpoint.md)		
- [torch.utils.cpp_extension](https://github.com/apachecn/pytorch-doc-zh/blob/master/docs/1.0/docs_cpp_extension.md)

>  \+ 326863350

**Comment by keyianpai** [comment]
[Torch Script](https://github.com/apachecn/pytorch-doc-zh/blob/master/docs/1.0/jit.md) + 2995452226

**Comment by univeryinli** [comment]
Distributed communication package - torch.distributed+844431684

**Comment by wizardforcel** [comment]
以下大佬们请尽快翻译：

@BreezeHavana @wangshuai9517 @sfyumi @JohnJiangLA @firdameng

**Comment by wangshuai9517** [comment]
> 以下大佬们请尽快翻译：
> 
> @BreezeHavana @wangshuai9517 @sfyumi @JohnJiangLA @firdameng

好的，过年过的忘了。两天左右弄好。

**Comment by JohnJiangLA** [comment]
> 以下大佬们请尽快翻译：
> 
> @BreezeHavana @wangshuai9517 @sfyumi @JohnJiangLA @firdameng

收到，这两天完成。

**Comment by hiyongz** [comment]
‘’torch‘’文档 我最近没时间翻译了，忙着写毕业论文，大佬把我撤了吧，非常抱歉！！@wizardforcel

**Comment by kunwuz** [comment]
kunwuz 514397511 torch.utils.dlpack torch.hub

**Comment by bruce1408** [comment]
Saving and Loading Models & Deep Learning for NLP with Pytorch + summer + 370926509

**Comment by luxinfeng** [comment]
luxinfeng + 1342839680 + Distributed communication package (deprecated) - torch.distributed.deprecated

**Comment by wizardforcel** [comment]
@JohnJiangLA @Lotayou @cloudyyyyy @WyattHuang1 @yiran7324 @yuange250 @qiaokuoyuan @luxinfeng

请加快进度谢谢

**Comment by cloudyyyyy** [comment]
> @JohnJiangLA @Lotayou @cloudyyyyy @WyattHuang1 @yiran7324 @yuange250 @qiaokuoyuan @luxinfeng
> 
> 请加快进度谢谢

十分抱歉，我应该没有太多时间翻译了，还请把我删除掉。

**Comment by Lotayou** [comment]
ditto

**Comment by gongel** [comment]
gongel+[torch.nn.LSTM](https://github.com/apachecn/pytorch-doc-zh/blob/master/docs/1.0/nn.md#lstm)+1324522527

**Comment by P3n9W31** [comment]
P3n9W31 + 348061475 + Custom C++ and CUDA Extensions

**Comment by sunxia233** [comment]
sunxia +871171307 +Extending TorchScript with Custom C++ Operators

**Comment by enningxie** [comment]
Chatbot Tutorial
+ 校验 + 1250557688

**Comment by luxinfeng** [comment]
Saving and Loading Models
What is torch.nn really?
Finetuning Torchvision Models
+校验+1342839680

**Comment by wizardforcel** [comment]
已释放两个月之前认领但没有翻译的章节

**Comment by enningxie** [comment]
Advanced: Making Dynamic Decisions and the Bi-LSTM CRF + 翻译 +1250557688

**Comment by bruce1408** [comment]
Reproducibility + summer + 370926509

**Comment by zonasw** [comment]
torch.optim + zonas + 602148598

**Comment by P3n9W31** [comment]
> P3n9W31 + 348061475 + Custom C++ and CUDA Extensions

进度状态需要更新哟。

**Comment by hijkzzz** [comment]
Generating Names with a Character-Level RNN
Classifying Names with a Character-Level RNN
校验+125036525

**Comment by wizardforcel** [comment]
@sunxia233 @enningxie @zonasw @bruce1408 你们四个的任务从翻译变成校对了，望悉知。

**Comment by TianFengshou** [comment]
Deep Learning for NLP with Pytorch
Introduction to PyTorch
Deep Learning with PyTorch
Deploying a Seq2Seq Model with the Hybrid Frontend
Sequence Models and Long-Short Term Memory Networks
Creating Extensions Using numpy and scipy
校验 +2404846224

**Comment by alexdyysp** [comment]
Data Loading and Processing Tutorial
校验 + 卿谦 + 2737502837 + 教程部分

**Comment by TianFengshou** [comment]
Deep Learning with PyTorch: A 60 Minute Blitz
What is PyTorch?
Autograd: Automatic Differentiation 
Neural Networks 
Training a Classifier
Optional: Data Parallelism

我看之前认领的人好像一直没动作，我想重新认领前六篇。

校验 +2404846224

**Comment by AllenZYJ** [comment]
Neural Transfer Using PyTorch
+校验+2533524298

**Comment by jiangzhonglian** [comment]
> Chatbot Tutorial
> 
> * 校验 + 1250557688

Chatbot Tutorial

校验 + 529815144 （我已经校验一个版本）

**Comment by alexdyysp** [comment]
Tensors
+卿谦+校验+2737502837+文档部分

**Comment by wizardforcel** [comment]
请加快速度：@gongel @sunxia233 @Smilexuhc @infdahai

**Comment by Originval** [comment]
（文档部分）torch.nn.functional + 校验 + 124638318

**Comment by alexdyysp** [comment]
torch.nn.init
+卿谦+校验+2737502837+文档部分

**Comment by wulongjian** [comment]
仙男 + 1726711669 + Learning PyTorch with Examples

**Comment by AlexJakin** [comment]
AlexJakin +  1272296763 + 校验 + Autograd mechanics

**Comment by AlexJakin** [comment]
AlexJakin + 1272296763 + 校验 + Broadcasting semantics

**Comment by AlexJakin** [comment]
AlexJakin + 1272296763 + 校验 + CUDA semantics

**Comment by jiangzhonglian** [comment]
1.2 版本: https://github.com/apachecn/pytorch-doc-zh/issues/422
